js/main.js

```diff
@@ -22,5 +40,62 @@ function formSubmitted(event) {
   data.nextEntryId++;
   data.entries.push(newEntry);
   $photo.src = 'images/placeholder-image-square.jpg';
+  $newEntryForm.setAttribute('class', 'entry-form hidden');
+  $entriesPage.setAttribute('class', 'entries');
   $formElement.reset();
+  if (newEntry.entryId === 1) {
+    journalFeedList.appendChild(creatingJournalEntry(newEntry));
+    $noEntriesText.setAttribute('class', 'no-entries-text hidden');
+  } else {
+    journalFeedList.insertBefore(creatingJournalEntry(newEntry), journalFeedList.firstChild);
+  }
+}
+
+function creatingDOMTree(tagName, attributes, children = []) {
+  var element = document.createElement(tagName);
+  for (var key in attributes) {
+    if (key === 'textContent') {
+      element.textContent = attributes.textContent;
+    } else {
+      element.setAttribute(key, attributes[key]);
+    }
+  }
+  for (var i = 0; i < children.length; i++) {
+    element.append(children[i]);
+  }
+  return element;
+}
+
+function creatingJournalEntry(newJournalEntry) {
+  var tree = creatingDOMTree('li', { class: 'entry-items' }, [
+    creatingDOMTree('div', { class: 'row' }, [
+      creatingDOMTree('div', { class: 'column-half' }, [
+        creatingDOMTree('img', { class: 'column-full remove-padding photo', alt: 'Some photo', src: newJournalEntry.photoUrl })
+      ]),
+      creatingDOMTree('div', { class: 'column-half' }, [
+        creatingDOMTree('h2', { class: 'entry-header', textContent: newJournalEntry.title }),
+        creatingDOMTree('p', { textContent: newJournalEntry.comment })
+      ])
+    ])
+  ]);
+  return tree;
+}
+
+function addingChildLoop(event) {
+  for (var i = localData.entries.length - 1; i >= 0; i--) {
+    if (localData.entries[i].entryId >= 1) {
```

Comment:
Don't rely on ids to know a count of items created. If a user had created an entry and then deleted it, this would lead to false positives. Instead, you should base the conditional on the value of the `length` property of `data.entries`